akka-http-core/src/test/scala/akka/http/impl/engine/client/TlsEndpointVerificationSpec.scala

```diff
@@ -47,6 +51,12 @@ class TlsEndpointVerificationSpec extends AkkaSpec("""
         e shouldBe an[Exception]
```

Comment:
The verifier is this: https://github.com/ktoso/ssl-config/blob/master/core/src/main/scala/com/typesafe/sslconfig/ssl/DefaultHostnameVerifier.scala